webpack/scenes/ContentViews/Details/Versions/Compare/CVVersionCompareTable.js

```diff
@@ -78,8 +79,15 @@ const CVVersionCompareTable = ({
           (
             <Tr key={`column-${result.id}`} ouiaId={`column-${result.id}`}>
               {columnHeaders.map(({ getProperty }, colIndex) =>
-                // eslint-disable-next-line react/no-array-index-key
-                <Td key={`cell-${colIndex}`}>{getProperty(result)} </Td>)}
+                (
+                  <Td
+                    className={(colIndex === columnHeaders.length - 3) ? 'border-right' : ''}
+                  // eslint-disable-next-line react/no-array-index-key
+                    key={`cell-${colIndex}`}
+                    style={(colIndex >= columnHeaders.length - 2) ? { paddingLeft: '2rem' } : {}}
```

Comment:
Stack overflow says this has a CSS equivalent.. 
```css
td:nth-last-child(-n+2)
```

Reply:
Updated to use td:nth-last-child for the borders.